barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.hpp

```diff
@@ -527,7 +527,6 @@ template <typename Curve> class ZeroMorphVerifier_ {
             if constexpr (Curve::is_stdlib_type) {
                 auto builder = x_challenge.get_context();
                 FF zero = FF(0);
-                zero.convert_constant_to_fixed_witness(builder);
```

Comment:
if the dummy_round is a witness we don't have to convert the constant into a witness because the result of conditional_assign I think will be a new witness as long as one of the values is not constant